pages/status.tsx

```diff
@@ -179,12 +181,38 @@ const Status: FC = () => {
             textRequired={t('common:required')}
             required
           />
-          <ActionButton
-            disabled={isCheckStatusLoading}
-            type="submit"
-            text={t('check-status')}
-            style="primary"
-          />
+          <div className="flex flex-wrap">
+            <div id="form-get-status" className="py-1 pr-2">
```

Comment:
I'm surprised moving the id fixed the cypress test... I don't think this id should have been moved; it should be an id for the whole form, not just the submit button.

Reply:
ahh right, maybe instead, the cy.get should target an id specifically on the button?

Reply:
Yeah, that would be the best solution.